packages/features/eventtypes/components/tabs/availability/EventAvailabilityTab.tsx

```diff
@@ -450,30 +438,38 @@ const TeamMemberSchedule = ({
         {isPending ? (
           <Spinner className="mt-2 h-6 w-6" />
         ) : (
-          <Controller
-            name={`hosts.${index}.scheduleId`}
-            render={({ field }) => {
-              return (
-                <Select
-                  placeholder={t("select")}
-                  options={options}
-                  isSearchable={false}
-                  onChange={(selected) => {
-                    field.onChange(selected?.value || null);
-                  }}
-                  className={classNames(
-                    "block w-full min-w-0 flex-1 rounded-sm text-sm",
-                    customClassNames?.select
-                  )}
-                  innerClassNames={customClassNames?.innerClassNames}
-                  value={value as AvailabilityOption}
-                  components={{ Option, SingleValue }}
-                  isMulti={false}
-                  isDisabled={isPending}
-                />
-              );
-            }}
-          />
+          <>
+            {!userHasDefaultSchedule ? (
+              <div className="my-2">
+                <Alert severity="warning" title={t("view_only_edit_availability_not_onboarded")} />
+              </div>
+            ) : (
+              <Controller
+                name={`hosts.${index}.scheduleId`}
+                render={({ field }) => {
+                  return (
+                    <Select
+                      placeholder={t("select")}
+                      options={options}
+                      isSearchable={false}
+                      onChange={(selected) => {
+                        field.onChange(selected?.value || null);
+                      }}
+                      className={classNames(
+                        "block w-full min-w-0 flex-1 rounded-sm text-sm",
+                        customClassNames?.select
+                      )}
+                      innerClassNames={customClassNames?.innerClassNames}
+                      value={value as AvailabilityOption}
+                      components={{ Option, SingleValue }}
+                      isMulti={false}
```

Comment:
![Screenshot 2024-11-30 230554](https://github.com/user-attachments/assets/f482864b-4db9-42ab-a21e-b3c50d99fd7c)

Show proper message instead of just disabled select. Earlier this would throw error and it was unhandled, replaced that